README.md

```diff
@@ -4,175 +4,225 @@
+requirements are minimal---as such, while `alloc` is still needed, this crate offers
 `no_std` support by omitting the default `std` feature.
 
-## Usage
+# Usage
+
 ```toml
 [dependencies]
 dcaf = { version = "^0.3" }
 ```
+
 Or, if you plan to use this crate in a `no_std` environment:
+
 ```toml
 [dependencies]
 dcaf = { version = "^0.3", default-features = false }
 ```
 
-## Example
+# Example
 
-As mentioned, the main features of this crate are ACE-OAuth data models and token creation/verification functions. We'll
-quickly introduce both of these here.
+As mentioned, the main features of this crate are ACE-OAuth data models and
+token creation/verification functions. We'll quickly introduce both of these here.
 
-### Data models
+## Data models
 
-[For example](https://www.ietf.org/archive/id/draft-ietf-ace-oauth-authz-46.html#figure-7), say you (the client) want to
-request an access token from an Authorization Server. For this, you'd need to create an `AccessTokenRequest`, which has
-to include at least a
-`client_id`. We'll also specify an audience, a scope (using `TextEncodedScope`---note that binary-encoded scopes
-or AIF-encoded scopes would also work), as well as a
-`ProofOfPossessionKey` (the key the access token should be bound to) in the `req_cnf` field.
+[For example](https://www.rfc-editor.org/rfc/rfc9200#figure-6),
+let's assume you (the client) want to request an access token from an Authorization Server.
+For this, you'd need to create
+an [`AccessTokenRequest`](https://docs.rs/dcaf/latest/dcaf/endpoints/token_req/struct.AccessTokenRequest.html), which
+has to include at least a
+`client_id`. We'll also specify an audience, a scope (
+using [`TextEncodedScope`](https://docs.rs/dcaf/latest/dcaf/common/scope/struct.TextEncodedScope.html)---note that
+[binary-encoded scopes](https://docs.rs/dcaf/latest/dcaf/common/scope/struct.BinaryEncodedScope.html)
+or [AIF-encoded scopes](https://docs.rs/dcaf/latest/dcaf/common/scope/struct.AifEncodedScope.html) would also work), as
+well as a
+[`ProofOfPossessionKey`](https://docs.rs/dcaf/latest/dcaf/common/cbor_values/enum.ProofOfPossessionKey.html) (the key
+the access token should be bound to) in the `req_cnf` field.
 
 Creating, serializing and then de-serializing such a structure would look like this:
+
 ```rust
 use dcaf::{AccessTokenRequest, ToCborMap, ProofOfPossessionKey, TextEncodedScope};
 
 let request = AccessTokenRequest::builder()
-   .client_id("myclient")
-   .audience("valve242")
-   .scope(TextEncodedScope::try_from("read")?)
-   .req_cnf(ProofOfPossessionKey::KeyId(base64::decode("6kg0dXJM13U")?))
-   .build()?;
+.client_id("myclient")
+.audience("valve242")
+.scope(TextEncodedScope::try_from("read") ? )
+.req_cnf(ProofOfPossessionKey::KeyId(base64::decode("6kg0dXJM13U") ? ))
+.build() ?;
 let mut encoded = Vec::new();
-request.clone().serialize_into(&mut encoded)?;
+request.clone().serialize_into( & mut encoded) ?;
```

Comment:
Is the indentation removal and the added whitespace around the `?`s and the `&` intentional?

Reply:
Oh, no it wasn't, thanks for pointing that out. I actually forgot to regenerate the README from the `lib.rs` crate-level docs, I'll do that now.

Reply:
Fixed in bdcb6ad917e5b63c51434feb6d449aed1b14d830.